src/client/auth/auth.proto

```diff
@@ -63,7 +63,21 @@ message ACL {
     map<string, Scope> entries = 1;
 }
 
+message User {
+    string username = 1;
+    bool admin = 2;
```

Comment:
Yeah, that's a potential problem, but like you noted it does save us a read on very API call.  Maybe the solution is to delete their token from etcd after someone's admin access has been revoked.